CHANGELOG.md

```diff
@@ -1,10 +1,18 @@
+## 2.14.0 (Feb, 3, 2022)
+FEATURES:
+
+* Added following smart remote repo attributes for npm, cargo, docker and helm remote repository resources [GH-305].
+  * "statistics_enabled"
+  * "properties_enabled"
+  * "source_origin_absence_detection"
+
 ## 2.13.1 (Unreleased)
```

Comment:
Can you update this with the release date? Thanks!

Reply:
done